src/storyblok-auth-api/reconcileNodeResponse.ts

```diff
@@ -10,11 +9,9 @@ import { InternalAdapter } from '../session-adapters/internalAdapter'
 export const reconcileNodeResponse = async ({
   res,
   responseElement,
-  adapter,
 }: {
   res: http.ServerResponse
   responseElement: ResponseElement
-  adapter: InternalAdapter
 }) => {
   if (responseElement.type === 'configuration-error') {
```

Comment:
Maybe we could even delete this method and just move the few lines to auth handler?

Reply:
that might be even clearer! i'm on it :)